romochka_bembot.js

```diff
@@ -56,11 +59,12 @@ const generateBemhtmlLink = bemhtml => {
  * @returns {String} Link
  */
 const generateBemjsonLink = bemjson => {
-	const trimedBemjson = _.trimEnd(bemjson, ';');
+	assert(typeof bemjson === 'string', 'Bemjson must be a string');
+
+	const trimedBemjson = bemjson.replace(/;+$/, '');
```

Comment:
@skad0 ahah) it's nice package) in that project it's ok because in visible future it will not grow up, so we can use this one little package) can you update PR?)